### PR TITLE
Setup for auto-assign

### PR DESCRIPTION
**Pull request by katiecrane**
### What does this PR do?

This PR includes the setup for [auto-assign](https://github.com/kentaro-m/auto-assign#usage)! The app has already been installed on GitHub and is being used for this repository.

### Any background context you want to provide?

@lindsayrmaher and I are out on Monday, so I included us anyway. @aaronschachter is out Monday-Wednesday so I left him off for now. We'll have to figure out how to handle for if folks are out or potentially swap to use 2 reviewers instead of 1, but I wanted to keep the system as simple as possible for our first pass 🤷‍♀ 

### What are the relevant tickets/cards?

🤠 🤖 

### Checklist

* [ ] Documentation added for new features/changed endpoints.
* [ ] Added appropriate feature/unit tests.
* [ ] Added screenshot to PR description of related front-end updates on **small** screens.
* [ ] Added screenshot to PR description of related front-end updates on **medium** screens.
* [ ] Added screenshot to PR description of related front-end updates on **large** screens.

### CODE REVIEW

File: .github/auto_assign.yml  [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#   - assigneeA
+
+# A number of assignees to add to the pull request
+# Set to 0 to add all of the assignees.
+# Uses numberOfReviewers if unset.
+# numberOfAssignees: 2
+
+# A list of keywords to be skipped the process that add reviewers if pull requests include it
+skipKeywords:
+  - wip
```
Comment (weerd) [inline .github/auto_assign.yml ]: Does is not ignore PRs in "Draft" mode?

**Comment by weerd** [comment]
Also what's the diff again between a "reviewer" and an "assignee"?

File: .github/auto_assign.yml  (hunk 1 above)
Comment (DFurnes) [reply .github/auto_assign.yml ]: I was wondering that too… I hope it does! 🤞

**Comment by katiecrane** [comment]
@weerd GREAT QUESTION about reviewer vs. assignee. I read the GitHub docs on each ([reviewer](https://help.github.com/en/articles/requesting-a-pull-request-review), [assignee](https://help.github.com/en/articles/assigning-issues-and-pull-requests-to-other-github-users)) and couldn't figure out the difference with respect to PRs based on that alone. Based on [this stack overflow post](https://stackoverflow.com/questions/41087206/on-github-whats-the-difference-between-reviewer-and-assignee) it seems that the "reviewer" is the person who reviews the code, and the "assignee" is the person in charge of merging in the PR once it is approved. I suppose that's useful on open source projects with random contributors, but doesn't really apply to us! Issues can also have "assignees" but the use case there is a lot more clear!

File: .github/auto_assign.yml  (hunk 1 above)
Comment (katiecrane) [reply .github/auto_assign.yml ]: I can't find any references to "draft" in the code, issues, or elsewhere in the repository so I believe it does NOT currently. BUT it is an open source project so if we really want that I'm sure we can make it happen. Initially though for our MVP test of this, I guess if you get assigned to a "draft" PR you get a pass until it is ready for review.

Comment (weerd) [reply .github/auto_assign.yml ]: 🤷‍♂ yeah, sounds good for now.

**Comment by weerd** [comment]
Ah, ok! Thanks for the follow up @katiecrane on "reviewer" vs "assignee" 🕵️‍♀️ So looks like for our needs just using "reviewers" and keeping "assignees" commented out in the config like you have it is ideal 💃

**Pull request merged**